src/master-playlist-controller.js

```diff
@@ -804,4 +814,42 @@ export default class MasterPlaylistController extends videojs.EventTarget {
       }
     });
   }
+
+  updateCues_(media) {
+    if (!this.useCueTags_ || !media.segments) {
+      return;
+    }
+
+    while (this.cueTagsTrack_.cues.length) {
+      this.cueTagsTrack_.removeCue(this.cueTagsTrack_.cues[0]);
+    }
+
+    let startTime;
+    let endTime = 0;
+
+    for (let i = 0; i < media.segments.length; i++) {
+      let segment = media.segments[i];
+
+      startTime = endTime;
+      endTime = startTime + segment.duration;
```

Comment:
The "short" value should be like 0.5s.
Also, we might want to consider making these have a start time of 0 and an end time of the cue time. That's what that w3c spec draft recommends for cuepoints.